article/app/controllers/LiveBlogController.scala

```diff
@@ -55,17 +55,24 @@ class LiveBlogController(
     }
   }
 
-  def renderArticle(path: String, page: Option[String] = None, filterKeyEvents: Option[Boolean]): Action[AnyContent] = {
+  def renderArticle(
+      path: String,
+      page: Option[String] = None,
+      filterKeyEvents: Option[Boolean],
+      automaticFilter: Option[String],
```

Comment:
wondering if this parameter could just be called filters? My feeling is "automatic" gives a bit too much information to the reader but I don't feel strongly about this

Reply:
Thanks for the suggestion @oliverlloyd I certainly like `topics` a lot more. Will make the change now 👍

Reply:
Note: needs to be changed here too:
https://github.com/guardian/fastly-edge-cache/pull/946

Reply:
Absolute pedant around naming here, but should this then filter down into the code e.g.
`topic=<enum>:<string> => TopMentionsTopic(type: TopicMentionsTopicType, value: String)`

Reply:
All done and also made a PR for fastly https://github.com/guardian/fastly-edge-cache/pull/948